src/containers/Home.jsx

```diff
@@ -34,6 +34,23 @@ class Home extends React.Component {
     orgLessUser: false
   }
 
+  handleOrgInviteClick = async (e) => {
+    e.preventDefault()
+    const newInvite = await this.props.mutations.createInvite({
+      id: 'cats',
```

Comment:
An empty object definitely doesn't work. {is_valid = true} does work, though - probably because that's the only required attribute in the model definition.